packages/fast-usdc-contract/src/fast-usdc.flows.ts

```diff
@@ -132,3 +154,226 @@ export const forwardFunds = async (
   }
 };
 harden(forwardFunds);
+
+export interface ContextAdvance {
+  chainHub: ChainHub;
+  feeConfig: FeeConfig;
+  getNobleICA: () => OrchestrationAccount<{ chainId: 'noble-1' }>;
+  log: Console['log'];
+  statusManager: StatusManager;
+  usdc: { brand: Brand<'nat'>; denom: Denom };
+  zcfTools: { makeEmptyZCFSeat: () => ZCFSeat };
+  zoeTools: ZoeTools;
+}
+
+export const advanceFunds = (async (
+  orch: Orchestrator,
+  {
+    chainHub,
+    feeConfig,
+    getNobleICA,
+    log,
+    statusManager,
+    usdc,
+    zcfTools,
+    zoeTools: { localTransfer, withdrawToSeat },
+  }: ContextAdvance,
+  { evidence, risk }: EvidenceWithRisk,
+  config: {
+    notifier: SettlerKit['notifier'];
+    borrower: LiquidityPoolKit['borrower'];
+    poolAccount: OrchestrationAccount<{ chainId: 'agoric-any' }>;
+    settlementAddress: CosmosChainAddress;
+  } & CopyRecord,
```

Comment:
why is `& CopyRecord` necessary?

Reply:
to meet the `Passable` constraint on things that go across the membrane.

Reply:
doesn't the type without `& CopyRecord` meet that?

Reply:
I see. I think that's a deficiency in the Exo or host membrane types because the literal satisfies CopyRecord,
```
  Types of parameters 'config' and 'args' are incompatible.
    Type 'Passable' is not assignable to type '{ notifier: Guarded<{ notifyAdvancingResult({ txHash, forwardingAddress, fullAmount, destination, }: { txHash: `0x${string}`; forwardingAddress: `noble1${string}`; fullAmount: NatAmount; destination: `${string}:${string}:${string}`; }, success: boolean): void; checkMintedEarly(evidence: CctpTxEvidence, destination: ...'.
```

out of scope but consider leaving an XXX or UNTIL https://github.com/Agoric/agoric-sdk/issues/9822